setup.cake

```diff
@@ -11,7 +11,7 @@ BuildParameters.SetParameters(
     repositoryName: "Cake.Yeoman",
     appVeyorAccountName: "cakecontrib");
 
-BuildParameters.PrintParamters(Context);
+BuildParameters.PrintParameters(Context);
```

Comment:
@gep13 Btw is there a reason why this is in each `setup.cake` file and not in the `build.cake` file inside Cake.Recipe?